velocity/tool/src/templates/permissions.vm

```diff
@@ -4,4 +4,4 @@
 	</h1>
 </div>
 
-<sakai-permissions tool="$!toolKey" />
\ No newline at end of file
+<sakai-permissions tool="$!toolKey" bundlekey="$!bundleKey" />
```

Comment:
Small thing, but I'd prefer you stick to the pattern used elsewhere of using hyphen to separate words, so bundle-key. You'll need to update the property in the component to be bundleKey: {attribute: "bundle-key", type: String}.

Reply:
I do like the bundle key change. Nice.

Reply:
Changes done. Thanks for the suggestion